packages/ui/components/form-core/src/choice-group/ChoiceGroupMixin.js

```diff
@@ -127,6 +127,13 @@ const ChoiceGroupMixinImplementation = superclass =>
       }
     }
 
+    /**
+     * @configure FocusMixin
+     */
```

Comment:
The FocusMixin is not applied to the ChoiceGroupMixin. 
It's usually applied to "endpoints", that only have one focusable element. The listbox for instance, that uses active-descendant, focuses either the el with role=listbox, or the textbox in case of a combobox. 

The ChoiceGroupMixin can be applied to:
- radiogroup
- checkboxgroup
- select/listbox/combobox (only in these cases the FocusMixin is applied)

Reply:
But maybe it's worth considering adding it indeed to the ChoiceGroupMixin 👍

Reply:
Or shall i apply these lines to the radio-group and checkbox-group?

Reply:
Atm the FocusMixin is only designed to have one focusable element (many of these cases would break if the second elm of a choicegroup received focus):
https://github.com/ing-bank/lion/blob/master/packages/ui/components/form-core/test/FocusMixin.test.js#L92

So then `_firstFocusablNode` would be a more appropriate name. 
Maybe we can try to solve it here for now? https://github.com/ing-bank/lion/blob/master/packages/ui/components/form/src/LionForm.js#L102